src/codegen.cpp

```diff
@@ -709,10 +709,11 @@ typedef struct {
 
 typedef struct {
     size_t len;
+    // variable length
     struct {
         int64_t isref;
         Function *f;
-    } data[];
+    } data[1];
```

Comment:
Doing the [1] thing can now give you runtime errors on OS X, cf  https://github.com/JuliaLang/julia/pull/13845